demo/code.py

```diff
@@ -1,4 +1,6 @@
+"""Demo code to show the LEDs working."""
 import time
+
```

Comment:
I assume this is PEP8 seperating the std lib from other 3rd party libraries. In circuit python "board" and "neopixel" are in the std lib